README.md

```diff
@@ -23,14 +23,149 @@ Or install it yourself as:
 
     $ gem install bitreserve
 
-## Usage
+# Contents
 
-TODO: Write usage instructions here
+* [1. Usage](#1-usage)
+* [2. Authentication](2#-authentication)
+  * [2.1 OAuth2](#21-oauth2)
+  * [2.2 Basic Auth](#21-basic-auth)
+  * [2.4 Personal Access Token](#24-personal-access-token)
+    * [2.4.1 Via argument](#241-via-argument)
+    * [2.4.2 Via environment variable](#242-via-environment-variable)
+* [3. Entities](#3-entities)
+* [4. API calls](#4-endpoints)
+  * [4.1 Currencies](#41-currencies)
+  * [4.2 Tickers](#42-tickers)
+  * [4.4 Entities](#43-entities)
+  * [4.4 Cards](#44-cards)
+  * [4.5 Transactions](#45-transactions)
+  * [4.6 Contacts](#46-contacts)
+  * [4.7 Users](#47-users)
+  * [4.8 Transparency](#48-transparency)
+* [5. Testing](X#-testing)
+* [6. Contributing](X#-contributing)
 
-## Contributing
+# 1 Usage
 
-1. Fork it ( https://github.com/[my-github-username]/bitreserve/fork )
+To start using the gem, you must instantiate the client
+
+```ruby
+client = Bitreserve::Client.new
+```
+
+All subsequent API calls are made from the client instance. For example:
+
+```ruby
+client.me
+# => #<Bitreserve::Entities::User:0x00000002cceff0>
+```
+
+# 2 Authentication
+
+## 2.1 OAuth2
+
+TODO
+
+## 2.2 Basic Auth
+
+TODO
+
+## 2.4 Personal Access Token
+
+If you already have a token, you can use by setting an environment variable, or by passing when instantiating the client
+
+### 2.4.1 Via argument
+
+Pass the token to the constructor:
+
+```ruby
+Bitreserve::Client.new(token: 'your-access-token')
+```
+
+### 2.4.2 Via environment variable
+
+Set the environment variable using [dotenv](https://github.com/bkeepers/dotenv), or by exporting it in your shell:
+
+```bash
+$ export BITRESERVE_AUTH_TOKEN="your-access-token"
```

Comment:
My guess is that we would eventually remove dotenv from the dependencies. It seems like an unnecessary dependency (a lot of people prefer to use it just for development, and not in production)

Especially since the preferred usage will be by passing the auth arguments to the constructor directly